### PR TITLE
m64p: Add version 2021.10.28

### LINKED ISSUE #300: Add m64p emulator (and others?)
I'd like to see [m64p](https://m64p.github.io/) (a Nintendo 64 emulator alternative to Project64) added as a bucket.

Here is also [a list of supposedly top rated emulators](https://www.reddit.com/r/emulation/comments/krllj2/2021_best_most_compatible_emulators_for/) by the Reddit community, some of which may be worth adding too.

### PR DESCRIPTION
**Pull request by satorope**
Make portable
Closes #300

### CODE REVIEW

**Review by Calinou — CHANGES_REQUESTED** [review]
The file needs to be renamed to `m64p.json` (instead of `m64p`) before this can be merged.

**Comment by satorope** [comment]
The name of the file has been modified.
Is this procedure okay?

**Review by Calinou — APPROVED** [review]
Thanks!

File: bucket/m64p.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "homepage": "https://m64p.github.io/",
+    "description": "Nintendo64 emulator bundled with paraLLEl-RDP for graphics",
```
Comment (Calinou) [inline bucket/m64p.json:3]:
```suggestion
    "description": "Nintendo 64 emulator bundled with paraLLEl-RDP for graphics",
```

**Pull request merged**